jenkins-slaves/ansible/data/service/smee.service.sh.j2

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export NODE_EXTRA_CA_CERTS=/etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.crt
```

Comment:
Maybe the export could be done in a profile (which would set it for the whole user environment), such as in [1]. But it can stay as it is as well.

[1] https://code.engineering.redhat.com/gerrit/gitweb?p=bxms-jenkins.git;a=blob;f=jenkins-image-extra-bits/rhba-osbs/ansible/roles/rhba-osbs/tasks/main.yml;h=710f2c16672c5511d05b60ce865b37d2e44ec03c;hb=HEAD#l115

Reply:
@winklerm I tried that but when the service was executed the `NODE_EXTRA_CA_CERTS` was not set

Reply:
OK